framework/src/meshgenerators/AdvancedExtruderGenerator.C

```diff
@@ -14,12 +14,14 @@
 #include "libmesh/function_base.h"
 #include "libmesh/cell_prism6.h"
 #include "libmesh/cell_prism18.h"
+#include "libmesh/cell_prism21.h"
```

Comment:
this reminds me that most of the changes you made for the other generators for TRI7 could also be done for PRISM21.

no need to if you dont need to use those

Reply:
Yes, that's what I asking above:

> 4. In AdvancedExtruderGenerator::generate() I'm building PRISM21s, so we might need to add formal support for these?

I'll add it to my list and if I have some time I'll do it later on. :)